### ISSUE TITLE
Onboarding Wizard should be pass accessibility standards

### ISSUE THREAD

**Issue by kjohnson** [impress-org/givewp#4957]
## User Story

As a site owner, I want my website to be accessible so that [some reason].

## Details

This is a grab bag issue (unfortunately), but there are a number of accessibility standards that need to be verified and/or addressed.

## Acceptance Criteria

-  [x] Elements must have sufficient [color contrast](https://dequeuniversity.com/rules/axe/3.5/color-contrast?application=AxeFirefox)
-  [x] Document must have [one main landmark](https://dequeuniversity.com/rules/axe/3.5/landmark-one-main?application=AxeFirefox)
- [x] Page must contain a [level-one heading](https://dequeuniversity.com/rules/axe/3.5/page-has-heading-one?application=AxeFirefox)
- [x] Elements should not have [tabindex](https://dequeuniversity.com/rules/axe/3.5/tabindex?application=AxeFirefox) greater than zero
- [x] id attribute value must be a [unique id](https://dequeuniversity.com/rules/axe/3.5/duplicate-id?application=AxeFirefox)
- [x] Form elements must have [labels](https://dequeuniversity.com/rules/axe/3.5/label?application=AxeFirefox)
